sw/airborne/firmwares/rotorcraft/stabilization/stabilization_attitude_quat_float.c

```diff
@@ -192,7 +192,9 @@ void stabilization_attitude_enter(void)
   /* reset psi setpoint to current psi angle */
   stab_att_sp_euler.psi = stabilization_attitude_get_heading_f();
 
-  attitude_ref_quat_float_enter(&att_ref_quat_f, stab_att_sp_euler.psi);
+  struct FloatQuat *state_quat = stateGetNedToBodyQuat_f();
+
+  attitude_ref_quat_float_enter(&att_ref_quat_f, state_euler);
```

Comment:
it should be `state_quat` as second argument

Reply:
Ah of course, no as you can see not tested yet :sweat_smile: 
I can test everything in simulation, will do that right away.